draft-id-claim-169-v1-1-0.md

```
@@ -0,0 +1,375 @@
+#### **Data formats**
+
+| Data format | Description |
+| ----------- | ----------- |
+| `0`         | Image       |
+| `1`         | Template    |
+| `2`         | Sound       |
+| `3`         | Bio hash    |
+
+#### **Data sub formats**
+
+**Image**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)